src/control-plane/backend/stack-action-state-machine-construct.ts

```diff
@@ -138,6 +138,7 @@ export class StackActionStateMachine extends Construct {
           'servicecatalog:DisassociateResource',
           'servicecatalog:TagResource',
           'servicecatalog:UntagResource',
+          'resource-groups:GetGroup',
```

Comment:
It does not work.

I added the below permissions to bypass the permission error.

```
                "resource-groups:GetGroup",
                "resource-groups:DisassociateResource",
                "tag:GetResources",
                "tag:UntagResources"
```

Ideally, we can restrict the resources with prefixes.

Reply:
I deployed the code with this change and it works. Let me double check.